swagger-spec/search/users.yaml

```diff
@@ -1 +1,68 @@
 # /search/users/
+get:
+  summary: Search the OSF for users
+  description: >-
+    This endpoint returns users that have been found by the given Elasticsearch query. All users returned
+    are serialized a user.
```

Comment:
Same typo - Either 'are serialized as users' or 'are serialized users' or 'are serialized' should be fine.